Testscripts/Windows/RemoveVhdxHardDisk.ps1

```diff
@@ -194,6 +194,12 @@ function Main {
         # Matches[1] represents the parameter name
         # Matches[2] is the value content of the parameter
         $controllerType = $Matches[1].Trim().ToLower()
+        if ($controllerType -like "scsi_*") {
+            $controllerType = "scsi"
+        }
+        elseif ($controllerType -like "ide_*") {
```

Comment:
} elseif ($controllerType -like "ide_*") {

Reply:
Updated, thank you for your comment.